__tests__/app/models/section.jest.js

```diff
@@ -0,0 +1,61 @@
+const db = require('../../../app/models/index')
+const moment = require('moment')
+
+describe("Section model", () => {
+
+    let section //declare section so it can be accessed in all the following tests, where necessary
+
+    beforeAll(async() => {
+        await db.sequelize.sync() // connect to the database
+    })
+
+    describe("Section.create", () => {
```

Comment:
And then here as well to test that it creates a unique one and can be accessed :)